CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ Based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), following [Se
 
 - Changed
   - Enabling `kube-linter` verbose output by default
+  - Upgrade Helm to 3.8.1
+  - Upgrade python to 3.10.3
+  - Upgrade chart-testing to 3.5.1
+  - Upgrade kubelinter to 0.2.5
```

Comment:
```suggestion
```

Reply:
Decided to not upgrade to the latest kube-linter version because it introduces additional checks that break peoples builds.